packages/expo-audio/src/ExpoAudio.ts

```diff
@@ -15,6 +15,10 @@ import { AudioPlayer, AudioRecorder } from './AudioModule.types';
 import { createRecordingOptions } from './utils/options';
 import { resolveSource } from './utils/resolveSource';
 
+const PLAYBACK_STATUS_UPDATE = 'onPlaybackStatusUpdate';
+const AUDIO_SAMPLE_UPDATE = 'onAudioSampleUpdate';
+const RECORDING_STATUS_UPDATE = 'onRecordingStatusUpdate';
```

Comment:
Let's remove `on` from the event names. `expo-video` doesn't use it.
I feel like it's ok for view events (so you can distinguish event callbacks from regular props), but not really needed for event emitter events.